cfme/automate/buttons.py

```diff
@@ -61,30 +119,42 @@ def __init__(self, text=None, hover=None, type=None, appliance=None):
         self.type = type
 
     def create(self):
-        navigate_to(self, 'Add')
+        view = navigate_to(self, 'Add')
+        view.fill({
+            'text': self.text,
+            'hover': self.hover,
+            'image': 'Button Image 1',
+        })
+        view.add.click()
```

Comment:
I am against such kind of names here. I would say `add_button` would be nice and readable.